scripts/depcheck/missing.ts

```diff
@@ -73,7 +73,7 @@ const IGNORE_MATCHES = {
     '@angular-devkit/core',
     '@angular-devkit/architect',
   ],
-  web: ['fibers', 'node-sass'],
+  web: ['@swc/core', 'fibers', 'node-sass'],
   workspace: [
```

Comment:
Might be worth calling out in a comment why this has an entry here (i.e. because of our decision to make it conditionally installed), you can see I tried to do this on line 38 when working on the lint conversion piece (similar but not identical case of dep being installed on demand).

I think without comments a lot of this file is not very approachable to anyone who didn't write it - 
"oh right we ignore that dep not being there in this case... why though? 🤔 " 😄